pkg/oc/cli/admin/release/image_mapper.go

```diff
@@ -190,7 +190,7 @@ func NopManifestMapper(data []byte) ([]byte, error) {
 // patternImageFormat attempts to match a docker pull spec by prefix (%s) and capture the
 // prefix and either a tag or digest. It requires leading and trailing whitespace, quotes, or
 // end of file.
-const patternImageFormat = `([\s\"\']|^)(%s)(:[\w][\w.-]{0,127}|@[A-Za-z][A-Za-z0-9]*(?:[-_+.][A-Za-z][A-Za-z0-9]*)*[:][[:xdigit:]]{2,})?([\s"']|$)`
+const patternImageFormat = `([\W]|^)(%s)(:[\w][\w.-]{0,127}|@[A-Za-z][A-Za-z0-9]*(?:[-_+.][A-Za-z][A-Za-z0-9]*)*[:][[:xdigit:]]{2,})?([\s"']|$)`
```

Comment:
It would be nice to split this pattern into several lines and document the matching rules. It makes parsing regexp easier from my own experience, and this one grew above regular parsing possibilities. Well, at least for me :wink: